server/api/api/endpoints/model_monitoring.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import fastapi
+from sqlalchemy.orm import Session
+
+import mlrun.common.schemas
+import server.api.utils.auth.verifier
+import server.api.utils.clients.chief
+from server.api.api import deps
+from server.api.api.endpoints.functions import process_model_monitoring_secret
+from server.api.crud.model_monitoring.deployment import MonitoringDeployment
+
+router = fastapi.APIRouter(prefix="/projects/{project}/model-monitoring")
+
+
+@router.post("/enable-model-monitoring")
+async def enable_model_monitoring(
+    project: str,
+    request: fastapi.Request,
```

Comment:
```suggestion
```
We used it for routing the request to the chief instance which is not longer required